tgui/packages/tgui/interfaces/CrewCredits/ReportTab.tsx

```diff
@@ -0,0 +1,100 @@
+import { useBackend, useLocalState } from "../../backend";
+import { Section, Stack, Tabs } from "../../components";
+import { PaperSheetView } from "../PaperSheet";
+import { CrewCreditsTabKeys, ReportData, ReportTabData } from "./type";
+
+export const ReportMenuTab = (props, context) => {
+  const { menu, setMenu } = props;
+  const { data } = useBackend<ReportTabData>(context);
+  const { reports } = data;
+  if (!reports.length) {
+    return;
+  }
+  return (
+    <Tabs.Tab
+      selected={menu === CrewCreditsTabKeys.Report}
+      onClick={() => setMenu(CrewCreditsTabKeys.Report)}>
+      Inspector&apos;s Report
+    </Tabs.Tab>
+  );
+};
+
+export const ReportTab = (props, context) => {
+  const [menu, setMenu] = useLocalState(context, 'issuersMenu', 0);
+  const { data } = useBackend<ReportTabData>(context);
+  const { reports } = data;
+  return (
+    <Section>
+      <Stack vertical>
+        <Stack.Item>
+          <Tabs>
+            {reports.map((report, index) => {
+              return (
+                <Tabs.Tab
+                  key={index}
+                  selected={menu === index}
+                  onClick={() => setMenu(index)}
+                >
+                  {report.issuer}
+                </Tabs.Tab>
+              );
+
+            })}
+          </Tabs>
+        </Stack.Item >
+        {reports.map((report, index) => {
+          return (
+            <Stack.Item key={index}>
+              { menu === index && <Report multi={index} {...report} />}
+            </Stack.Item>
+          );
+
+        })}
+
+      </Stack>
+    </Section>
+  );
+};
+
+const Report = (props: ReportData, context) => {
+  const [menu, setMenu] = useLocalState(context, 'pageMenu', "0-0");
+  const { issuer, pages, multi } = props;
+  return (
+    <Stack vertical>
+      <Stack.Item>
+        <Section title={issuer}>
+          <Tabs>
+            {pages.map((page, index) => {
+              const multi_index = `${multi}-${index}`;
+              return (
+                <Tabs.Tab
+                  key={multi_index}
```

Comment:
You don't need to worry about the `key` of different reports being the same, `key` only has to be unique at the layer it's applying to. That is to say: I think you can get away with stripping out that multi prefix.